defaults/network-ents.go

```diff
@@ -1,6 +1,7 @@
 package defaults
 
 import (
+	"fmt"
```

Comment:
I think `gofmt` just only enforces alphabetical order for each block, but does not reorganize into multiple blocks